packages/sommelier.rb

```diff
@@ -3,23 +3,23 @@
 class Sommelier < Package
   description 'Sommelier works by redirecting X11 programs to the built-in ChromeOS Exo Wayland server.'
   homepage 'https://chromium.googlesource.com/chromiumos/platform2/+/HEAD/vm_tools/sommelier/'
-  version '20210109-1'
+  version '20210109-2'
```

Comment:
I think this package wins the award for the most recompiled package. :1st_place_medal: